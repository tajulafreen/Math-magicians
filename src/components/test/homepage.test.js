import React from 'react';
import { render } from '@testing-library/react';
import Home from '../homepage';
import '@testing-library/jest-dom/extend-expect';

describe('Home component', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
