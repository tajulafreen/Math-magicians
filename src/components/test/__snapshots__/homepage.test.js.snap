// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match the snapshot 1`] = `
<div>
  <div
    class="home"
  >
    <h3>
      Welcome to our page!
    </h3>
    <p>
      Welcome to our calculator app! We are delighted to have you join our community of number enthusiasts. Whether you are a student, professional, or simply someone who loves crunching numbers, our app is designed to be your reliable companion in all your mathematical endeavors. With a sleek and user-friendly interface, our calculator app offers a wide range of functions to help you solve complex equations, perform quick calculations, and explore the fascinating world of mathematics. From basic arithmetic operations to advanced trigonometric functions, logarithms, and more, our app has got you covered. Not only does our calculator app provide accurate and speedy calculations, but it also offers additional features that enhance your experience. You can customize the app appearance, switch between light and dark modes, and even save your favorite calculations for future reference. But that is not all. We understand that learning and discovering new mathematical concepts can be a thrilling journey, so we have included a dedicated section with educational resources, tips, and tricks to expand your mathematical prowess. We are committed to providing you with a seamless and enjoyable calculator experience. Our team of developers is constantly working to improve the app, adding new features and addressing any concerns that may arise. Your feedback is invaluable to us, so please do not hesitate to reach out and share your thoughts. Thank you for choosing our calculator app. Get ready to embark on a mathematical adventure like never before!
    </p>
  </div>
</div>
`;
